XboxKeyboardMouse/XboxController/Emulation/XboxStream.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Drawing;
+using System.Linq;
```

Comment:
This can be removed

Reply:
It gave me the convenient [].Contains but maybe I could have used List<string> for a similar result

Reply:
Never mind - this is fine. I thought it was an unused import.